pom.xml

```diff
@@ -70,15 +71,26 @@
 
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.1</version>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+        <version>${junit.jupiter.version}</version>
       <scope>test</scope>
-    </dependency> 
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>${junit.jupiter.version}</version>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>com.epimorphics</groupId>
       <artifactId>appbase</artifactId>
-      <version>3.1.14</version>
+      <version>4.0.0-SNAPSHOT</version>
```

Comment:
Update to 4.0.0

Reply:
Done.

Reply:
Why? 

This is normally released through mvn release so the release will be 4.0.0 but at this stage the pom should say 4.0.0-SNAPSHOT surely. Think I'm missing something.